pkgs/servers/mail/exim/default.nix

```
@@ -92,11 +92,11 @@ stdenv.mkDerivation rec {
       done )
   '';
 
-  meta = {
+  meta = with stdenv.lib; {
     homepage = "http://exim.org/";
     description = "A mail transfer agent (MTA)";
-    license = stdenv.lib.licenses.gpl3;
-    platforms = stdenv.lib.platforms.linux;
-    maintainers = [ stdenv.lib.maintainers.tv ];
+    license = with licenses; [ gpl3 ];
```

It's a single license, so no need to use a list. Also, the NOTICE file included with exim states that the software falls under the GPLv2 or later.

```suggestion
    license = licenses.gpl2Plus;
```

---

Hm, it also seems to have this: https://github.com/Exim/exim/blob/master/src/LICENSE.opendmarc in case DMARC support is included.